models/profileDiffs.js

```diff
@@ -0,0 +1,30 @@
+const firestore = require('../utils/firestore')
+const profileDiffsModel = firestore.collection('profileDiffs')
+
+/**
+ * Fetches the pending profile diffs
+ * @param query { Object }: Filter for profile diffs data
```

Comment:
It seems the parameter is not required. Can we please remove it?

Reply:
Done removed it